### DISCUSSION TITLE
Improved book reading

### DISCUSSION THREAD

**Discussion by KrakenSly26** [jellyfin/jellyfin#8610] [General] [answered]
Lors de la lecture de Comics ou de livres le fond d'écran est Blanc, ce qui est désagréable pour les yeux, serait-il possible d'afficher un fond noir ou de donner le choix à l'utilisateur final. D'avance merci de votre réponse.

**Comment by nielsvanvelzen** [comment, marked as answer]
Feature requests go to https://features.jellyfin.org
